src/test/scala/com/gu/zuora/creditor/CreditTransferServiceTestMixin.scala

```diff
@@ -0,0 +1,30 @@
+package com.gu.zuora.creditor
+
+import java.util.concurrent.atomic.AtomicInteger
+
+import com.gu.zuora.creditor.holidaysuspension.CreateCreditBalanceAdjustment
+
+trait CreditTransferServiceTestMixin {
```

Comment:
Could this just be an object?

Reply:
i actually move them to private functions as the are used only in `CreditTransferServiceTest`